pkg/api/v1/types.go

```diff
@@ -2031,7 +2031,6 @@ type ServiceSpec struct {
 
 	// This service will route traffic to pods having labels matching this selector.
 	// Label keys and values that must match in order to receive traffic for this service.
-	// If empty, all pods are selected, if not specified, endpoints must be manually specified.
```

Comment:
"if not specified, endpoints must be manually specified" <--- don't remove this part.